Makefile

```diff
@@ -25,9 +25,16 @@ release-container: docker-running
 	docker rm -f cockpit-release-stage || true
 	docker run --privileged --name=cockpit-release-stage \
 		--entrypoint=/usr/local/bin/Dockerfile.sh cockpit/infra-release:staged
+	$(eval tag := cockpit/infra-release:$(shell date --iso-8601))
```

Comment:
May I suggest moving this line outside of the rule, above the rule and using an upper case variable name? Typically make variables are upper case:

```
TAG := cockpit/infra-release:$(shell date --iso-8601)

release-container:
       ...
```